CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 10.17.6 - 2019-07-23
+
+* Update mixpanel tracking [Paulo Castro]
+
```

Comment:
> You'll also need a CHANGELOG entry

Done, thanks. This will however only appear in the `v10` branch, not `master` (master of course already has a commit for this: v11.7.3).

Reply:
Yes, that's fine. You can annotate the entry as `[BACKPORT] Update [...]` as well to make it more obvious to anyone looking at v10 CHANGELOG.

Reply:
Done